packages/kbn-config-schema/src/index.ts

```diff
@@ -102,7 +101,7 @@ function maybe<V>(type: Type<V>): Type<V | undefined> {
 }
 
 function nullable<V>(type: Type<V>): Type<V | null> {
-  return new NullableType(type);
+  return schema.oneOf([type, schema.literal(null)], { defaultValue: () => null });
```

Comment:
__nit:__ can it just be `defaultValue: null`?

Reply:
I think I tried that first, it didn't work, and you suggested the function invocation version, which did work.  Let me try again ...

Reply:
Huh, only reference I can find is you suggesting this change a long time ago to me.  Weird thing is, no idea where the function version came from, I don't think I would have figured that one out myself; I have a feeling maybe it was needed in a previous version of the combinator.

So, commit incoming with that change ...

Reply:
Heh, yeah, I believe that was me who initially suggested that function syntax :see_no_evil: Just didn't think long enough to suggest the simplest solution, sorry for the confusion.